man/newgidmap.1.xml

```diff
@@ -88,11 +88,11 @@
     <title>DESCRIPTION</title>
     <para>
       The <command>newgidmap</command> sets <filename>/proc/[pid]/gid_map</filename> based on its
-      command line arguments and the gids allowed. The subid delegation can come either from files
-      (<filename>/etc/subgid</filename>) or from the configured NSS subid module. Only one of them
-      can be chosen at a time. So, for example, if the subid source is configured as NSS and
-      <command>groupadd</command> is executed, then the command will fail and the entry will not be
-      created in <filename>/etc/subgid</filename>.
+      command line arguments and the gids allowed. Subgid delegation can either be managed via
+      <filename>/etc/subgid</filename> or through the configured NSS subid module. These options
+      are mutually exclusive.
+      Note, that <command>groupadd</command> will only create entries in
```

Comment:
Shall this go to ` man/subgid.5.xml`?

Reply:
Yes, I have already updated the group files to be consistent with the changes in the user files.